src/pages/workspaces/workspace/analysis/modals/ComputeModal.test.js

```diff
@@ -742,6 +742,9 @@ describe('ComputeModal', () => {
       const customImageUri = 'us'
       await fireEvent.change(imageInput, { target: { value: customImageUri } })
 
+      const customImageTimeoutLimit = await screen.findByText('Creation Timeout Limit')
+      expect(customImageTimeoutLimit).toBeInTheDocument()
```

Comment:
findByText will throw if not exactly one instance of text is found.  So this toBeInTheDocument check is not needed.  Throwing to fail the test is sufficient, and has the benefit of outputting helpful DOM debug info